wordpress/wp-content/plugins/memberful-wp/js/memberful-wp-tinymce-plugin.js

```diff
@@ -72,43 +72,53 @@
   }
 
   function insertCheckoutLinkDialog(editor, options) {
-    var checkoutItemCtrl = {}, linkTextCtrl = {};
-
     function optionsForPurchasables(current) {
       return {text: current.name, value: current.slug};
     };
 
-    options = options || {};
-
-    checkoutItemCtrl = {
+    var checkoutItemCtrl = {
       name: "item",
       type: "listbox",
       label: options.label,
       values: options.choices.map(optionsForPurchasables)
     };
 
-    linkTextCtrl = {
+    var linkTextCtrl = {
       name: "linkText",
       type: "textbox",
       label: "Link text"
     };
 
+    var body = [checkoutItemCtrl, linkTextCtrl];
+
+    if(options.showPrice) {
+      var priceCtrl = {
+        name: "price",
+        type: "textbox",
+        label: "Price (optional)",
+        tooltip: "Works only with plans where members are allowed to set price."
```

Comment:
What do you think of using the "choose what you pay" terminology here? Admins might not associate "allowed to set price" with CWYP.

Maybe something like _"Works only with plans where members can choose what they pay"_?

Reply:
I like that! I changed it in b5cede474ddb166278b770c9225a951ffe01180c.

@okdan Are you OK with the copy suggested by Lucas?

Reply:
That works for me 👍